shared/store/configure-store.js

```diff
@@ -60,10 +60,16 @@ if (closureStoreCheck) {
   middlewares.push(closureCheck)
 }
 
-const createStoreWithMiddleware = applyMiddleware.apply(null, middlewares)
-
 export default function configureStore (initialState: any) {
-  const s = configureStoreNative(createStoreWithMiddleware)(rootReducer, initialState)
+  const store = createStore(rootReducer, initialState, storeEnhancer(middlewares))
+
+  if (module.hot) {
+    // $FlowIssue
+    module.hot.accept('../reducers', () => {
```

Comment:
Does this work on mobile? Unfortunately RN uses a different API for `module.hot.accept`. See https://webpack.github.io/docs/hot-module-replacement.html#accept vs https://facebook.github.io/react-native/blog/2016/03/24/introducing-hot-reloading.html#hmr-api :disappointed: